src/api/salesApi.js

```diff
@@ -0,0 +1,22 @@
+const response = {
+    data: {
+        sales: {
+            finalDate: new Date(2018, 5, 1),
+            personalizationDate: new Date(2017, 9, 15),
+            exactQuantityDate: new Date(2017, 9, 15),
+            campus: 150,
+            online: 419,
+            max: 1500
+        }
+    }
+};
+
+export function fetchSales(){
+    console.log('[API] fetchSales');
+    return new Promise((resolve, reject) => {
+        setTimeout(() => {
+            resolve(Object.assign({}, response.data));
```

Comment:
You don't need return new reference here, becouse you do it in salesReducer.js. btw. data fetched from API always will be have new reference.